src/styles/blocks/nav.scss

```
@@ -0,0 +1,47 @@
+.nav {
+  &__list {
+    display: flex;
+    align-items: center;
+  }
+
+  &__item {
+    list-style: none;
+    margin-right: 20px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+    text-transform: uppercase;
+    text-decoration: none;
+    display: flex;
+    align-items: center;
+    @extend %font-500;
+
+    height: 60px;
```

```suggestion
    display: flex;
    align-items: center;
    height: 60px;
        
    @extend %font-500;    
    text-transform: uppercase;
    text-decoration: none;
```

Follow [this order](https://codeguide.academy/html-css.html#css-order) when setting properties to css-element.